src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -761,6 +764,9 @@ internal int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancella
                                     testSymWriterFactory: null,
                                     diagnostics: diagnosticBag,
                                     metadataOnly: emitOptions.EmitMetadataOnly,
+                                    includePrivateMembers: emitOptions.IncludePrivateMembers,
```

Comment:
The `/refout` command-line scenario should pass `EmitMetadataOnly` = `false`, `IncludePrivateMembers` = `fase` and `refPeStreamProviderOpt` (secondary output stream) set.

I think your proposal would break that scenario, resulting in `includePrivateMembers` = `true` passed into this method call.